src/Umbraco.Web/HtmlHelperRenderExtensions.cs

```diff
@@ -243,9 +240,9 @@ protected override void Dispose(bool disposing)
                     || _controllerName == "UmbProfile"
                     || _controllerName == "UmbLoginStatus"
                     || _controllerName == "UmbLogin")
-			    {
+                {
                     _viewContext.Writer.Write(AntiForgery.GetHtml().ToString());
-			    }
+                }
 
                 //write out the hidden surface form routes
                 _viewContext.Writer.Write("<input name='ufprt' type='hidden' value='" + _encryptedString + "' />");
```

Comment:
Just a side note: the single quotes should probably be changed to double quotes...

Reply:
Happy for that change if you have time